src/components/Activity/styles.js

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+
+export const Card = styled.article`
+    display: inline-flex;
+    flex-direction: column;
+    flex-grow: 1;
+    justify-content: center;
+    align-items: center;
+    width: 22.5em;
```

Comment:
```suggestion
    width: 22.5rem;
```
Amg muda pf todos as medidas de `em` pra `rem` pf, pq em eh uma medida relativa, enquanto rem é a medida a partir da medida raiz